randovania/games/fusion/logic_database/Sector 2 TRO.txt

```diff
@@ -584,7 +584,9 @@ Extra - room_id: [18]
   > Door to Zazabi Access
       After Boss Zazabi Defeated and Can Climb High Jump Wall
   > Door to Zazabi Speedway
-      Screw Attack and Space Jump and After Boss Zazabi Defeated
+      All of the following:
+          Screw Attack and After Boss Zazabi Defeated
+          Space Jump or Can Single Walljump
```

Comment:
the `can single walljump` should be accomanied by a WJ trick. This is **not** trivial to do.